docs/flatcar-linux/kvm-libvirt.md

```diff
@@ -0,0 +1,297 @@
+# KVM libvirt
+
+In this tutorial, we'll create a Kubernetes cluster with Flatcar Linux inside VMs on our local machine.
+
+We'll declare the cluster using the Lokomotive Terraform module for the KVM libvirt platform.
+
+Controllers are provisioned to run an `etcd-member` peer and a `kubelet` service. Workers run just a `kubelet` service. A one-time [bootkube](https://github.com/kubernetes-incubator/bootkube) bootstrap schedules the `apiserver`, `scheduler`, `controller-manager`, and `coredns` on controllers and schedules `kube-proxy` and `calico` (or `flannel`) on every node. A generated `kubeconfig` provides `kubectl` access to the cluster.
+
+## Requirements
+
+* free RAM (minimum 4 GB)
+* libvirtd and a user who is allowed to create virtual machines
```

Comment:
I would move the permissions to a separate line and explain more explicitly what is required. Example:

* libvirtd running (see [some-section][some-section] for installation instructions)
* Permissions to access `/dev/kvm` (see [another-section][another-section] for more info)